indico/modules/rb/client/js/components/App.jsx

```diff
@@ -86,6 +86,7 @@ class App extends React.Component {
     }
 
     onSidebarHide = () => {
+        document.body.style.overflowY = 'auto';
```

Comment:
🙈 but I guess it's needed as a workaround.

Could we use a class for it and add/remove it (using `document.body.classList`)? Like this it's clear that this and the line where the style attribute is set to hidden are related (since searching for the class name will show all relevant places in the code)